.travis.yml

```diff
@@ -1,14 +1,12 @@
 language: node_js
+sudo: false
```

Comment:
A tweak to allow Travis to start up faster. More info [here](http://docs.travis-ci.com/user/workers/container-based-infrastructure/)